src/util.rs

```diff
@@ -147,14 +143,14 @@ fn sockaddr_to_network_addr(sa: *const libc::sockaddr) -> (Option<MacAddr>, Opti
             let mac = MacAddr((*sll).sll_addr[0], (*sll).sll_addr[1],
                               (*sll).sll_addr[2], (*sll).sll_addr[3],
                               (*sll).sll_addr[4], (*sll).sll_addr[5]);
-            return (Some(mac), None);
+            (Some(mac), None)
```

Comment:
When using implicit return, could you put an empty line before it, please? eg.

```
some_code;

return_val
```

Rather than:

```
some_code;
return_val
```